imports/Flags.ts

```diff
@@ -1,5 +1,4 @@
 import { Match, check } from 'meteor/check';
-import { SHA256 } from 'meteor/sha';
 import FeatureFlags from './lib/models/FeatureFlags';
 
 const Flags = {
```

Comment:
Maybe also drop the `shard?` argument on the next line too, as it's now unused?

Reply:
Oh good catch. I'll clean that up